# print(f'2 + 3 = {2 + 3}')
# то же самое, только коротко
print(f'{2 + 3 = }')
print(f'{9 - 12 = }')
print(f'{15 * 2 = }')
print(f'{3 / 4 = }')
print(f'{20 // 6 = }')
print(f'{20 % 6 = }')
print(f'{11 ** 2 = }')
print(f'{81 ** (1/2) = }\n')

print(f'{2.0 + 3 = }')
print(f'{9.0 - 12 = }')
print(f'{15.0 * 2 = }')
print(f'{3.0 / 4 = }')
print(f'{20.0 // 6 = }')
print(f'{20.0 % 6 = }')
print(f'{11.0 ** 2 = }')
print(f'{81.0 ** (1/2) = }\n')

print(f"{'abc' + 'xyz' = }")
# приведёт к возникновению исключения TypeError
# print(f"{'12' + 5 = }")
print(f"{'abc' * 3 = }")

